### ISSUE TITLE
[v6] Reducer plugin is beeing called for any form

### ISSUE THREAD

**Issue by olegkalyta** [erikras/redux-form#1127]
I wrote plugin for one form
```js
const formWithPlugin = reduxFormReducer.plugin({
    publicProcessEdit: handleActions({
        'redux-form/CHANGE': (state, { meta: { field }, payload: value }) => {
            // ...
        })
    })
})
```
But this plugin is also called for any other form

Additionally, it occurs error when i try to update value in plugin:
```js
return update(state, { values: { repeatEvery: { value: { $set: 1 } } } })
```
because it is called for right form 1 time, and another form in second time for wrong form `state` is empty and i am getting:

```
Can't change value of undefined
```

My temporary solution for it in plugin:
```js
if (Object.getOwnPropertyNames(state).length === 0) {
  return state
}
```

**Comment by erikras** [comment]
Ha! 🙈 Testing _FAIL_ on my part. Good catch. Well reported.

**Comment by jjduhamel** [comment]
+1

**Comment by erikras** [comment]
This is fixed. I will comment here again when the fix is published.

**Comment by erikras** [comment]
Published in [`v6.0.0-rc.1`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.1).
